doc/whats_new.rst

```diff
@@ -349,6 +354,7 @@ People who contributed to this release (in alphabetical order):
 - `Mainak Jas`_
 - `Nick Tolley`_
 - `Ryan Thorpe`_
+- `Rajat Partani`_
```

Comment:
don't add your name here. This is the list of people who contributed to the 0.1 release. We're now working on the 0.3 release ... we'll make this list when the time comes.

Reply:
Sure will remove

Reply:
Done @jasmainak